concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -1198,7 +1237,6 @@ jobs:
               - -e
               - -c
               - |
-                export CONCOURSE_URL="https://${CONCOURSE_HOSTNAME}.${SYSTEM_DNS_ZONE_NAME}"
```

Comment:
This has to remain as it was. This job needs to always connect to the deployed concourse. With this change, it fails when run on the bootstrap concourse as it tried to connect to `localhost:8080` which fails.